dotnet3.5/S3/DeleteBucketExample/DeleteBucket/DeleteBucket.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.DeleteBucket]
+
+using Amazon;
+using Amazon.S3;
+using System;
+using System.Threading.Tasks;
+
+namespace DeleteBucket
+{
+    class DeleteBucket
+    {
+        // This example shows how to delete an existing and empty bucket.
+        //  The examples uses AWS .NET SDK 3.5 and .NET 5.0
+
+        // Change the name of the following constant to the AWS Region containing your bucket
+        // The value below is just an example.
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the name of the bucket to delete.
+        private const string BUCKET_NAME = "doc-example-bucket";
+
+        static void Main()
+        {
+            _s3Client = new AmazonS3Client(BUCKET_REGION);
+
+            // Now delete the bucket. If the bucket you are trying to
+            // delete contains any objects, the call will raise an exception.
+            Console.WriteLine($"\nDeleting bucket {BUCKET_NAME}...");
+            DeletingBucketAsync(_s3Client, BUCKET_NAME).Wait();
+        }
+
+        /// <summary>
```

Comment:
Missing summary, parameter, and return descriptions.

Reply:
I noticed that when I started fixing errors and checked all documentation comments.